uint8 msgCnt
string id
int16 secMark
int64 lat
int64 longitude
int16 elev
PositionalAccuracy_t[] accuracy
int32 transmission
int16 speed
int16 heading
int8 angle
AccelerationSet4Way_t[] accelSet
BrakeSystemStatus_t[] brakes
VehicleSize_t[] size

